mama/c_cpp/src/c/reservedfields.c

```diff
@@ -114,7 +111,7 @@ void initReservedFields (void)
 {
     CREATE_FIELD (MsgType,            U8);
     CREATE_FIELD (MsgStatus,          U8);
-    CREATE_FIELD (FieldIndex,         U16);
+    CREATE_FIELD (FieldIndex,         VECTOR_U16);
```

Comment:
Why? Weird field too never noticed it before and can't find any trace of it being used - expands to MamaReservedField**Field**Index. Is this necessary?